core/src/apps/benfen/layout.py

```
@@ -0,0 +1,54 @@
+) -> bool:
+    from trezor.strings import strip_amount
+
+    result = await should_show_details(
+        ctx,
+        title=_(i18n_keys.TITLE__SEND_MULTILINE).format(
+            strip_amount(format_benfen_amount(value, currency_symbol))[0]
+        ),
+        address=to_addr,
+        br_code=ButtonRequestType.SignTx,
+    )
+    return result
```

Ruff (0.8.2)

Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)